### ISSUE TITLE
Option to change fonts

### ISSUE THREAD

**Issue by rtofvnt** [microsoft/vscode-serial-monitor#162]
Would it be possible to add an option to control the font face?
![image](https://github.com/microsoft/vscode-serial-monitor/assets/7382936/470a3f3d-6a72-4884-af90-b70ded0e3e0a)

**Comment by Eplankton** [comment]
Good idea!

**Comment by MicSG-dev** [comment]
This new functionality would be great.

**Comment by gcampbell-msft** [comment]
@rtofvnt @Eplankton @MicSG-dev Thank you all for the request. Currently, we match the font of the `editor.fontFamily` setting from VS Code. Therefore, you should be able to change this setting and see it matched in the Serial Monitor output. 

Are you requesting a different setting specific to the Serial Monitor? Or does the current implementation work for you?

**Comment by Eplankton** [comment]
> @rtofvnt @Eplankton @MicSG-dev Thank you all for the request. Currently, we match the font of the `editor.fontFamily` setting from VS Code. Therefore, you should be able to change this setting and see it matched in the Serial Monitor output.
> 
> Are you requesting a different setting specific to the Serial Monitor? Or does the current implementation work for you?

Yes, we mean that a separate setting options to change font in serial monitor, perhaps `serial.fontFamily`, by now we already have the option to change the size/weight of font in the monitor, but sometime just want another font.

**Comment by gcampbell-msft** [comment]
@Eplankton Thanks for the follow-up! I've placed it on the backlog and we will monitor both our workload as well as community demand for it. 

Thank you!

**Comment by Eplankton** [comment]
> @Eplankton Thanks for the follow-up! I've placed it on the backlog and we will monitor both our workload as well as community demand for it.
> 
> Thank you!

Which part of code is responsible for the font style? I would like to modify by myself :)
